contracts/CourtPresaleActivate.sol

```diff
@@ -161,10 +167,10 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
     * @dev User cannot specify minimum output or deadline.
     */
     function () external payable {
-        _contributeEth(1, block.timestamp);
```

Comment:
Ahh I missed this in #3, but I wonder if we should add a check for `msg.value > 0`?

Reply:
But this is done in the internal function.